packages/spektate/package.json

```diff
@@ -8,6 +8,7 @@
     "build": "shx rm -rf lib && tsc -p . && webpack",
     "change": "beachball change",
     "check": "beachball check",
+    "lint": "tslint -p tsconfig.json -c tslint.json src/**/*.ts",
```

Comment:
we `spk` have recently move away from tslint and use eslint because tslint is deprecated. Do we want to use eslint too?

Reply:
@dennisseah yeah we should open a low-pri task for that, for now I added this because no linting was in place. Could either remove this or let it be added until that task is worked on. Thoughts?

Reply:
i am leaning towards merging this PR and switch to eslint later if it requires a lot of code fixing.

Reply:
Okay, will open an issue for that